Added the markdown file containing information on the getCalendarType… (#3025)
Codecademy/docs: content/java/concepts/calendar/terms/getCalendarType/getCalendarType.md

---
Title: '.getCalendarType()'

> Description: 'Returns the calendar type of the Calendar instance'

Reviewer: ```suggestion
Description: 'Returns the calendar type of the Calendar instance.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Calendar'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The `.getCalendarType()` method of the `Calendar` class is used to return a String type of the calendar used.

Reviewer: ```suggestion
The **`.getCalendarType()`** method of the `Calendar` class is used to return a string representation of the calendar used.
```

## Syntax

```pseudo

> 
> public String getCalendarType()
> 

Reviewer: ```suggestion
calendarInsta.getCalendarType()
```

```

> 

Reviewer: Generally, there should be a sentence here that explains the syntax. In some cases it will be minimal, but it should address the parameters(if there are any); special cases; return values or any other notes.

## Example

The following example displays the type of the calendar.

```java

import java.util.*;

public class MyClass {
  public static void main(String[] args) {

>     //creating a Calendar object with specified date

Reviewer: ```suggestion
    // Creating a Calendar object with specified date
```

Reviewer: Comments should begin with a capital and there should be a space before the first word

    Calendar Cal = Calendar.getInstance();

    //printing the Calendar
    System.out.println("The Calendar is: " + Cal.getTime());

    //getting the calendar type
    System.out.println("Calendar Type: " + Cal.getCalendarType());
  }
}

```

> This will output the following:

Reviewer: ```suggestion
This will output a result similar to the following:
```

Reviewer: The other option is to specify a specific date instead of using the current time in the example.

```shell

> 
> The calendar is: <Current DAY MONTH TIME (HH:MM:SS) UTC YEAR> (For eg., Wed Aug 09 10:00:00 UTC 2023)
> The calendar type: gregory
> 

Reviewer: ```suggestion
 Wed Aug 09 10:00:00 UTC 2023
The calendar type: gregory
```
